### PR TITLE
Update RegisterVaultView.swift

### PR DESCRIPTION
**Pull request by realpaaao**
@Rockindash my bad of not updating the figma label. This should be the label.

### CODE REVIEW

File: VultisigApp/VultisigApp/Views/Vault/RegisterVaultView.swift [hunk 1]
```diff
@@ -67,7 +67,7 @@ struct RegisterVaultView: View {
     }
     
     var label: some View {
-        FilledButton(title: "complete")
+        FilledButton(title: "Save Vault QR")
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Views/Vault/RegisterVaultView.swift:70]: You need to create an item in `Localizable.strings` , and use the name here , so the text is translated into other languages

Comment (Rockindash) [reply VultisigApp/VultisigApp/Views/Vault/RegisterVaultView.swift]: Updated the localizable files for all the languages.

**Review by johnnyluo — APPROVED** [review]
LGTM

**Comment by johnnyluo** [comment]
@Rockindash can you resolve the conflict in this PR , after that , you can merge it

**Pull request merged**